.rspec

```diff
@@ -0,0 +1,4 @@
+--require spec_helper
```

Comment:
You don't need to require spec_helper when you are requiring the rails_helper

Reply:
spec_helper has been removed.